content/applications/services/helpdesk/timesheet_and_invoice/reinvoice_from_project.rst

```diff
@@ -12,15 +12,15 @@ to accurately bill the client.
 Configuration
 =============
 
-To turn on the :guilabel:`Timesheet on Ticket` feature, go to :menuselection:`Helpdesk -->
-Configuration --> Helpdesk Teams`, select a Helpdesk team, and then, click :guilabel:`Edit`. Next,
-enable the options :guilabel:`Timesheet on Ticket` and :guilabel:`Time Reinvoicing`. Then, click
-:guilabel:`Save` to apply these changes to the Helpdesk team settings.
+To turn on the :guilabel:`Timesheets` feature, go to :menuselection:`Helpdesk --> Configuration
+--> Helpdesk Teams`, select a Helpdesk team, and then, click :guilabel:`Edit`. Next, enable the
+options :guilabel:`Timesheets` and :guilabel:`Time Billing`. Then, click :guilabel:`Save` to apply
+these changes to the Helpdesk team settings.
 
-Once the :guilabel:`Timesheet on Ticket` feature is enabled, a :guilabel:`Project` drop-down option
-will appear below :guilabel:`Timesheet on Ticket`. The :guilabel:`Project` drop-down option will
-automatically be set to a Project that Odoo created for the Helpdesk team. The tickets' timesheets
-will be stored in the selected Project.
+Once the :guilabel:`Timesheets` feature is enabled, a :guilabel:`Project` drop-down option will
+appear below :guilabel:`Timesheets`. The :guilabel:`Project` drop-down option will automatically
+be set to a :guilabel:`Project` that Odoo created for the Helpdesk team. The tickets' timesheets
```

Comment:
Before I go ahead and commit changes, a question for clarification: should I also make `Project` lowercase then? Since I'm not talking about a GUI object or referencing a specific project.